content/articles/domains-cm.md

```diff
@@ -0,0 +1,28 @@
+---
+title: .CM Domains
+excerpt: This article explains the requirements and special procedures for .CM domain names.
+categories:
+- Domains
+---
+
+# .CM Domain Names
+
+* TOC
+{:toc}
+
+---
+
+This article explains the requirements and special procedures for .CM domain names.
+
+
+## Domain Restrictions on a .CM domain {#domain-restrictions}
+
+If the name servers of a .CM domain are modified more than three to four times in a month, the domain is set to serverUpdateProhibited for the next four weeks.
```

Comment:
I know this is the language they used, but would it be clearer to say "a 30-day period" than "a month"?

Reply:
I have no personal preference. CNR also doesn't specify the length of a "month" here.